### PR TITLE
#135 Repair gateway receive attributes return value

### PR DESCRIPTION
**Pull request by felix-ma**
Repair gateway receive attributes return value

### CODE REVIEW

**Comment by felix-ma** [comment]
Repairing the problem that telemetry data can not be transmitted singleton

File: src/main/java/org/thingsboard/gateway/util/converter/BasicJsonConverter.java [hunk 1]
```diff
@@ -77,6 +77,10 @@ protected DeviceData parseDeviceData(DocumentContext document) throws ParseExcep
             for (TransformerKVMapping mapping : mappings) {
                 String key = eval(document, mapping.getKey());
                 String strVal = eval(document, mapping.getValue());
+                if(mapping.getValue().equals(strVal)){
+                    //取值错误，跳过这个数据
```
Comment (mp-loki) [inline src/main/java/org/thingsboard/gateway/util/converter/BasicJsonConverter.java]: Please provide comment in English or remove it

File: src/main/java/org/thingsboard/gateway/util/converter/BasicJsonConverter.java [hunk 2]
```diff
@@ -104,6 +108,10 @@ protected DeviceData parseDeviceData(DocumentContext document) throws ParseExcep
...
```
Comment (mp-loki) [inline src/main/java/org/thingsboard/gateway/util/converter/BasicJsonConverter.java]: Please provide comment in English or remove it

File: src/main/java/org/thingsboard/gateway/service/gateway/MqttGatewayService.java [hunk 3]
```diff
@@ -657,8 +656,8 @@ private MqttClient initMqttClient() {
 
             tbClient.on(DEVICE_ATTRIBUTES_TOPIC, this).await(connection.getConnectionTimeout(), TimeUnit.MILLISECONDS);
             tbClient.on(DEVICE_GET_ATTRIBUTES_RESPONSE_PLUS_TOPIC, this).await(connection.getConnectionTimeout(), TimeUnit.MILLISECONDS);
-            tbClient.on(DEVICE_GET_ATTRIBUTES_RESPONSE_PLUS_TOPIC, this).await(connection.getConnectionTimeout(), TimeUnit.MILLISECONDS);
 
+            tbClient.on(GATEWAY_RESPONSES_ATTRIBUTES_TOPIC, this).await(connection.getConnectionTimeout(), TimeUnit.MILLISECONDS);
```
Comment (mp-loki) [inline src/main/java/org/thingsboard/gateway/service/gateway/MqttGatewayService.java]: Why is this space needed?

Comment (felix-ma) [reply src/main/java/org/thingsboard/gateway/service/gateway/MqttGatewayService.java]: 659 blank lines are used to differentiate gateway topic and device topic.
If you don't need it, I can delete it.

File: src/main/java/org/thingsboard/gateway/util/converter/BasicJsonConverter.java (hunk 1 above)
Comment (felix-ma) [reply src/main/java/org/thingsboard/gateway/util/converter/BasicJsonConverter.java]: Revised

File: src/main/java/org/thingsboard/gateway/util/converter/BasicJsonConverter.java (hunk 2 above)
Comment (felix-ma) [reply src/main/java/org/thingsboard/gateway/util/converter/BasicJsonConverter.java]: Revised

**Comment by felix-ma** [comment]
Has been modified to annotate in English.

**Comment by ashvayka** [comment]
LGTM

**Pull request merged**